examples/run/run.cpp

```diff
@@ -21,15 +24,43 @@
 #include "json.hpp"
 #include "llama-cpp.h"
 
-#define printe(...)                   \
-    do {                              \
-        fprintf(stderr, __VA_ARGS__); \
-    } while (0)
+#if defined(_WIN32)
+#    define FORMAT_ATTR(fmt, args)
+#else
+#    define FORMAT_ATTR(fmt, args) __attribute__((format(printf, fmt, args)))
+#endif
```

Comment:
You could also use `GGML_ATTRIBUTE_FORMAT` from `ggml.h`.

Reply:
Done